protoc-gen-openapiv2/internal/genopenapi/template.go

```diff
@@ -117,31 +117,52 @@ func listEnumNames(reg *descriptor.Registry, enum *descriptor.Enum) (names []str
 		}
 		names = append(names, value.GetName())
 	}
-	return names
+
+	if len(names) > 0 {
+		return names
+	}
+
+	return nil
```

Comment:
I don't think this needs to change

Reply:
Generated openapi docs would contains `"enum": []` when remove this change, which is not the expected behavior before.

Reply:
🤔 that's wild, but okay.